registration/locale/pl/LC_MESSAGES/django.po

```diff
@@ -284,35 +298,35 @@ msgstr "hasło (ponownie)"
 
 #: templates/registration/password_change_done.html:7
 msgid "Password successfully changed!"
-msgstr ""
+msgstr "Udało się zmienic hasło!"
 
 #: templates/registration/password_change_form.html:4
 #: templates/registration/password_change_form.html:10
 #, fuzzy
 #| msgid "password"
 msgid "Change password"
-msgstr "hasło"
+msgstr "Zmień hasło"
 
 #: templates/registration/password_reset_complete.html:4
 msgid "Password reset complete"
-msgstr ""
+msgstr "Hasło zresetowane"
 
 #: templates/registration/password_reset_complete.html:8
 msgid "Your password has been reset!"
-msgstr ""
+msgstr "Twoje hasło zostaało zresetowane!"
 
 #: templates/registration/password_reset_complete.html:9
 #, python-format
 msgid "You may now <a href=\"%(login_url)s\">log in</a>"
-msgstr ""
+msgstr "Możesz teraz <a href=\"%(login_url)s\">się zalogować</a>"
 
 #: templates/registration/password_reset_confirm.html:10
 msgid "Confirm password reset"
-msgstr ""
+msgstr "Potwierdź reset hasła"
 
 #: templates/registration/password_reset_confirm.html:14
 msgid "Enter your new password below to reset your password:"
-msgstr ""
+msgstr "Wprowaðx ponizej nowe hasło, by zresetować swoje hasło"
```

Comment:
"Wprowaðx" ?

Reply:
Fixed. Next time feel free to correct it yourself. It won't take you longer than commenting ;)

Reply:
Fixed.